lib/entities/biometric_auth.dart

```diff
@@ -1,32 +1,32 @@
-import 'package:local_auth/local_auth.dart';
 import 'package:flutter/services.dart';
-import 'package:cake_wallet/generated/i18n.dart';
+import 'package:flutter_local_authentication/flutter_local_authentication.dart';
 
 class BiometricAuth {
-  final _localAuth = LocalAuthentication();
+  final _flutterLocalAuthenticationPlugin = FlutterLocalAuthentication();
 
   Future<bool> isAuthenticated() async {
     try {
-      return await _localAuth.authenticate(
-          localizedReason: S.current.biometric_auth_reason,
-          options: AuthenticationOptions(
-            biometricOnly: true,
-            useErrorDialogs: true,
-            stickyAuth: false));
+      final authenticated = await _flutterLocalAuthenticationPlugin.authenticate();
+      return authenticated;
     } on PlatformException catch (e) {
       print(e);
     }
-
     return false;
   }
 
   Future<bool> canCheckBiometrics() async {
+    bool canAuthenticate;
     try {
-      return await _localAuth.canCheckBiometrics;
-    } on PlatformException catch (e) {
-      print(e);
+      canAuthenticate = await _flutterLocalAuthenticationPlugin.canAuthenticate();
+
+      // Setup TouchID Allowable Reuse duration
+      // It works only in iOS and macOS, but it's safe to call it even on other platforms.
+      await _flutterLocalAuthenticationPlugin.setTouchIDAuthenticationAllowableReuseDuration(30);
+    } on Exception catch (error) {
```

Comment:
I believe this is not needed, and you can also remove the specific `on Exception` so that it catches any error even if Dart considers the error not a descendant from the `Exception` parent class